src/lib/profiles/network-provisioning/NetworkInfo.h

```diff
@@ -53,24 +53,26 @@ class NL_DLL_EXPORT NetworkInfo
     NetworkInfo();
     ~NetworkInfo();
 
-    ::nl::Weave::Profiles::NetworkProvisioning::NetworkType
-          NetworkType;              /**< The type of network. */
-    int64_t NetworkId;              /**< The network id assigned to the network by the device, -1 if not specified. */
+    /** The type of network. */
```

Comment:
This was an example where the existing infix notation did not work well.  I think the infix notation could have been fixed if we did not use fully qualified types, but as is the `type varName;` was just too long, especially with the `type` spilling into the infix doxygen. I'd be happy to refactor this back to infix with the shortened types, I think it is possible.